guardian/prosemirror-typerighter
# Add 'document changed' indicator

## What does this change?

Currently a user has no way of knowing if Typerighter should run again, given that the document has changed in a significant way. This could mislead a user into thinking a document has been checked when it hasn't.

This PR adds a small blue dot to indicate when a document has changed since the last time the 'Check document' button was pressed.

## How to test
Run the branch locally and make a change to the document. See the blue dot appear. Click 'Check document' and see it disappear until the document is changed again.

## How can we measure success?
Users are able to tell if the matches shown are as up-to-date as possible.

## Have we considered potential risks?
Users might not realise what the dot is for, or might find it irritating.

## Images
![doc_changed](https://user-images.githubusercontent.com/15648334/96151007-dd2d0b00-0f02-11eb-9a8a-6fd3f7ae5f52.gif)
